yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -405,7 +405,7 @@ func (ar *AssessmentReport) CalculateSizeDetails() (SizeDetails, error) {
 			} else {
 				details.TotalTableSize += utils.SafeDereferenceInt64(stat.SizeInBytes)
 				details.TotalTableRowCount += utils.SafeDereferenceInt64(stat.RowCount)
-				if slices.Contains(colocatedTables, fmt.Sprintf("%s.%s", stat.SchemaName, stat.ObjectName)) {
+				if slices.Contains(colocatedTables, stat.ObjectName) {
```

Comment:
But in case of PG we need qualified so will have to add handling as per source here

Reply:
Yeah..good point. Made the change. 
I think we went off in the wrong direction here. I'm not satisfied with this PR, but going ahead in the interest of time. The change for removing schemas should ideally happen at the html layer. and the struct/json should have remained intact. 
We should pick this up post release.

Reply:
Yes, right